app/Friend.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Friend extends Model
```

Comment:
Same as with group, I think we should just represent Friends as `User` with Friend table joining two user `id`'s